### PR TITLE
UW-627: Adding a test Jupyter notebook

### PR DESCRIPTION
**Pull request by Byrnetp**
**Synopsis**

We are adding an example Jupyter notebook to test using a pull request.

**Type**

- [ ] Bug fix (corrects a known issue)
- [ ] Code maintenance (refactoring, etc. without behavior change)
- [ ] Documentation
- [x] Enhancement (adds new functionality)
- [ ] Tooling (CI, code-quality, packaging, revision-control, etc.)

**Impact**

- [ ] This is a breaking change (changes existing functionality)
- [x] This is a non-breaking change (existing functionality continues to work as expected)

**Checklist**

- [ ] I have added myself and any co-authors to the PR's _Assignees_ list.
- [x] I have reviewed the documentation and have made any updates necessitated by this change.

### CODE REVIEW

File: docs/notebooks/dictToYAML.ipynb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "22e97042-de70-4a04-81fd-dabadab9ba3c",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```
Comment (maddenp-noaa) [inline docs/notebooks/dictToYAML.ipynb:49]:
```suggestion
  }
```
We probably don't need to commit this empty cell.

Comment (maddenp-noaa) [reply docs/notebooks/dictToYAML.ipynb:49]: Alternatively, I think you could reject this suggestion it the GitHub web interface, delete the empty cell in the notebook in your browser, save it, then commit and push the change -- which should accomplish the same thing, but maybe it's safer to make notebook changes in the notebook interface iteslf.

Comment (Byrnetp) [reply docs/notebooks/dictToYAML.ipynb]: Good catch! I needed to make other changes in the notebook interface, so I made this change there as well.

**Review by christinaholtNOAA — APPROVED** [review]
LGTM. We might consider a different name for the notebook in the future, but I think this is fine for now.

**Review by maddenp-noaa — APPROVED** [review]
Looks good. I'm not sure if you will be able to merge the PR given that you're not a member of the `ufs-community` group or an official contributor to the repo. If you don't see that option, please let us know and one of us will do the merge.

**Comment by NaureenBharwaniNOAA** [comment]
I missed UW Group yesterday and I just wanted to check in to see if there is a link to the jupyter notebook to test running the cells? The code itself looked accurate to me, but wasn't sure how to run the actual example notebook. It could be I'm just missing a connection from yesterdays meeting.

**Pull request merged**